libs/order-list/src/lib/components/order-data-provider/order-data-provider.ts

```diff
@@ -79,8 +79,8 @@ export const prepareIncomingOrders = (delta: OrderFields[]) => {
   return incoming;
 };
 
-const update = (data: OrderFields[], delta: OrderFields[]) =>
-  produce(data, (draft) => {
+const update = (data: OrderFields[], delta: OrderFields[]) => {
+  return produce(data, (draft) => {
```

Comment:
chore (non-blocking): any reason for preferring a function body + explicit return? seen a few changes like this in the PR, got me curious :)

Reply:
This was requested by Matt, he prefer `"arrow-body-style": ["error", "always"]`